app/views/devise/sessions/new.html.slim

```diff
@@ -12,16 +18,21 @@ h1.h3.my0 = decorated_session.new_session_heading
   = f.input :password, label: t('account.index.password'), required: true
   = f.input :request_id, as: :hidden, input_html: { value: request_id }
   = hidden_field_tag 'ga_client_id', '', class: 'ga-client-id'
-  div
-    = f.button :submit, t('links.next'), class: 'sm-col-6 col-12 btn-wide mb3'
-
-p.my3
-  = link_to t('notices.terms_of_service.link'), MarketingSite.privacy_url, target: '_blank'
+  div.mb3
+    = f.button :submit, t('links.next'), class: 'col-12 btn-wide mb2'
+    .create-box.bg-light-blue.center.py1.rounded-lg
+      = t('links.create_helper')
+      = link_to t('links.create_account'), sign_up_email_url(request_id: request_id), \
+        class: 'pl1'
 
 .clearfix.pt1.border-top
```

Comment:
What is a way to get this to render conditionally? on the plain sign in (not from an SP) it's an empty div but it has some spacing that it's leaving behind so it bumps the forgot your password link down a little lower than it should be.

Reply:
How about an: `- unless request_id` similar to the `- if request_id` up top.

Reply:
Went with decorated_session.sp_name since request param could drop off and still be in session